app/liquid_tags/email_subscription_tag.rb

```diff
@@ -0,0 +1,9 @@
+class UserSubscriptionTag < LiquidTagBase
+  def initialize(_tag_name, cta_text, _tokens)
+    @cta_text = cta_text.strip
+  end
+
+  def render(context); end
+end
+
+Liquid::Template.register_tag("user_subscription", UserSubscriptionTag)
```

Comment:
This entire liquid tag will be built in the next PR. For now, it's "empty" so that
```ruby
liquid_tags.include?(UserSubscriptionTag)
```
doesn't blow up in the controller with `undefined`.